commands.cpp

```diff
@@ -1256,6 +1273,34 @@ int add_gpg_user (int argc, const char** argv)
 		new_files.push_back(state_gitattributes_path);
 	}
 
+	// Add the merge-tool script to the repo state directory to provide "git merge" command
+	const std::string		state_mergetool_path(state_path + "/merge-tool.sh");
+	if (access(state_mergetool_path.c_str(), F_OK) != 0) {
+		std::ofstream		state_mergetool_file(state_mergetool_path.c_str());
+		//                          |--------------------------------------------------------------------------------| 80 chars
+		state_mergetool_file << "#!/usr/bin/env bash\n";
```

Comment:
Please use `/bin/sh` instead of `/usr/bin/env bash` for better compatibility – not every system ships bash by default. `/bin/sh` exists on every *nix system, it’s defined by POSIX, and this script is compatible with POSIX shell.